packages/ui/src/loading-indicator/loading-indicator.spec.ts

```diff
@@ -25,6 +25,22 @@ describe('LoadingIndicator', () => {
         expect(parent.firstChild).toMatchSnapshot();
     });
 
+    it('renders loading indicator with provided styles prop', () => {
+        indicator = new LoadingIndicator({
+            styles: {
+                size: 10,
+            },
+        });
+
+        indicator.show(parentId);
+
+        const child = parent.firstChild as HTMLElement;
```

Comment:
Can we avoid using `as` here? Seems like we are adding the eslint rule just to bypass this condition check?

Reply:
Yes, I added extra rule only for spec.ts files. We have the same rule added in several other packages.

Reply:
I don't think that we should spend extra time to avoid using `as` here

Reply:
We have that rule in other packages since we added eslint last month, and to fix all the issues is a major task. 
But the goal going forward is to avoid adding rule overrides.

Reply:
Is it possible to skip it for now? I will update this tests and remove extra eslint rules right after removing paypal commerce implementation from code package

Reply:
```
function isHtmlElement(value?: HTMLElement): value is HTMLElement {
    return value !== undefined && value instanceof HTMLElement;
}
```

should be similar to this.

Reply:
I found a way how to make it without typeguard. Thanks!